[exporter/datasetexporter]: Initial implementation of logs and traces (#21815)
open-telemetry/opentelemetry-collector-contrib: exporter/datasetexporter/examples/e2e/README.md

# E2E Example

> This is simple example that demonstrates how to use open telemetry collector with `DataSetExporter`.

Reviewer: ```suggestion
This is a simple example that demonstrates how to use the OpenTelemetry collector with `DataSetExporter`.
```

## How To Use It

1. Modify [otel-config.yaml](otel-config.yaml)
   1. Update `dataset_url` with the URL to your dataset server
   2. Update `api_key` with your API key
2. Run `docker-compose up --abort-on-container-exit`

## What It Does

It is using `telemetrygen` to generate logs and traces. Dataset exporter is defined there twice:

> 1. It's defined for accepting logs. It has 

Reviewer: This seems incomplete.

Author: Fixed. I am sorry about it. 🙈
